hazelcast/src/main/java/com/hazelcast/map/impl/operation/PartitionWideEntryBackupOperation.java

```diff
@@ -44,10 +45,16 @@ public void run() {
         responses = new MapEntries(recordStore.size());
         EntryOperator operator = operator(this, backupProcessor, getPredicate(), true);
 
-        Iterator<Record> iterator = recordStore.iterator(Clock.currentTimeMillis(), true);
+        Iterator<Record> iterator = recordStore.loadAwareIterator(Clock.currentTimeMillis(), true, true);
```

Comment:
This will actually make this the only 2 entry operations that don't call the get interceptors before getting the values.
Also, if we choose to use the values from the iterator, I guess the HD partition wide operations might be changed as well.